typescript/sdk/src/providers/transactions/submitter/submitterBuilderGetter.ts

```diff
@@ -171,6 +170,10 @@ export async function getSubmitter<TProtocol extends ProtocolType>(
     ...defaultSubmitterFactories,
   };
 
+  for (const protocol of altVmSigner.getSupportedProtocols()) {
+    mergedSubmitterRegistry[protocol] = MULTI_VM_SUBMITTERS_FACTORIES;
```

Comment:
There is an inconsistency here between `.getSupportedProtocols()` - which is only implemented in the CLI - and `MULTI_VM_SUBMITTERS_FACTORIES` - which is defined in this file. If we just initialise the `defaultSubmitterFactories` as before, we can avoid passing the `altVmSigner` here altogether.

Later on (after this PR), we can rearrange the code so that the knowledge about which protocols are implemented doesn't reside in the CLI.